operations/helm/charts/mimir-distributed/values.yaml

```
@@ -3779,6 +3784,23 @@ metaMonitoring:
     #  capabilities:
     #    drop: [ALL]
 
+    # -- Node selector for Deployment Pods
+    nodeSelector: {}
+    # -- Tolerations for Deployment Pods
+    tolerations: []
+
+    # -- topologySpreadConstraints allows to customize the default topologySpreadConstraints.
+    # More info: https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/#topologyspreadconstraints-field
+    topologySpreadConstraints:
+      - maxSkew: 1
+        topologyKey: kubernetes.io/hostname
+        whenUnsatisfiable: ScheduleAnyway
+        minDomains: 1
+        nodeAffinityPolicy: Honor
+        nodeTaintsPolicy: Honor
+        matchLabelKeys:
+          - pod-template-hash
```

We use this as a default constraint for other workloads in this file - is there a reason not to do the same here?

```suggestion
      - maxSkew: 1
        topologyKey: kubernetes.io/hostname
        whenUnsatisfiable: ScheduleAnyway
        # minDomains: 1
        # nodeAffinityPolicy: Honor
        # nodeTaintsPolicy: Honor
        # matchLabelKeys:
        #   - pod-template-hash
```